plan/property_cols_prune.go

```diff
@@ -78,6 +78,14 @@ func (p *LogicalJoin) preparePossibleProperties() [][]*expression.Column {
 }
 
 func (la *LogicalAggregation) preparePossibleProperties() [][]*expression.Column {
-	la.possibleProperties = la.children[0].preparePossibleProperties()
+	// `preparePossibleProperties` is a recursive method. So children's `preparePossibleProperties` must be called.
+	childProps := la.children[0].preparePossibleProperties()
+	// If there's no group-by item, the stream aggregation could have no order property. So we can add an empty property
+	// when its group-by item is empty.
+	if len(la.GroupByItems) == 0 {
+		la.possibleProperties = [][]*expression.Column{nil}
```

Comment:
Is it possible that `childProps` is not nil but `len(la.GroupByItems) == 0`?

Reply:
If so, can we skip `la.children[0].preparePossibleProperties()`?

Reply:
`la.children[0].preparePossibleProperties()` should be called in whatever situations.